azuredeploy.json

```diff
@@ -359,6 +359,24 @@
             },
             "type": "string"
         },
+        "nfsOnCtlrVmDiskStorageType": {
+            "type": "string",
+            "defaultValue": "Premium_LRS",
+            "allowedValues": [
+                "Premium_LRS",
+                "Standard_LRS"
+            ],
+            "metadata": {
+                "description": "Azure storage type for NFS Disks created on controller VM. Applicable only when parameter 'fileServerType' has value 'nfs'."
+            }
+        },
+        "nfsOnCtlrVmDiskSizeGB": {
```

Comment:
fileServerDiskSize parameter already exists for this purpose

Reply:
Done, thanks.